java.base/src/main/java/jdk/internal/misc/Signal$_native.java

```diff
@@ -0,0 +1,43 @@
+package jdk.internal.misc;
+
+import org.qbicc.runtime.Build;
+
+class Signal$_native {
+
+    private static int findSignal0(String sigName) {
+        if (Build.Target.isPosix()) {
+            switch(sigName) {
+                case "HUP": return 1;
+                case "INT": return 2;
+                case "QUIT": return 3;
+                case "ILL": return 4;
+                case "ABRT": return 6;
+                case "FPE": return 8;
+                case "KILL": return 9;
+                case "SEGV": return 11;
+                case "PIPE": return 13;
+                case "ALRM": return 14;
+                case "TERM": return 15;
+                default: return -1;
```

Comment:
These signal numbers *should* come from constants in `org.qbicc.runtime.posix.Signal` and `org.qbicc.runtime.stdc.Signal`.

Reply:
Fixed & squashed.